src/RegistrationController.js

```diff
@@ -147,7 +143,7 @@ function (
           var data = Okta.Model.prototype.toJSON.apply(this, arguments);
           return {
             userProfile: data,
-            relayState: self.getRelayStateData()
+            relayState: this.settings.get('relayState')
```

Comment:
same comment here, we are not supporting the url relayState. Are we guaranteeing we are not breaking any flow? I rather have the url as a second option (look at my other comment)